src/header.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './header.css';
+
+const Header = () => {
+  return (
+    <header className='header'>
+      <h1>Rancid Taters!</h1>
+    </header>
+  )
+}
+
+export default Header;
```

Comment:
Perfect, if we decided later to add a login with specific users we could easily turn this functional component into a class component that stores the user as state and displays their name.